stylesheet/components/product.css

```diff
@@ -0,0 +1,70 @@
+.sm-container{
+          min-width: 1080px;
+          margin-top: -40rem;
+          margin-left: 280px;
+          height: 60rem;
+     
+     }
+
+.row-2{
+     justify-content: space-between;
+     margin: 50px  130px  ;
+    }
+
+    select{
+         border: 1px solid var(--primary);
+    }
+    select:focus{
+         outline: none;
+    }
+
+    .page-btn{
+         margin-top: 3rem;
+         margin-right: 50rem;
+
+         /* margin: 0 auto 40px;
+     */
+    }
+    .page-btn span{
+         display: inline-block;
+         border:  1px solid var(--primary);
+         margin-left: 10px;
+         width: 40px;
+         height: 40px;
+         text-align: center;
```

Comment:
this property can be move the util CSS like txt-center class

Reply:
okay